src/core/lib/security/security_connector/load_system_roots_linux.cc

```diff
@@ -51,20 +51,27 @@ GPR_GLOBAL_CONFIG_DEFINE_STRING(grpc_system_ssl_roots_dir, "",
 namespace grpc_core {
 namespace {
 
-const char* kLinuxCertFiles[] = {
+#if defined(GPR_LINUX) || defined(GPR_ANDROID)
+const char* kCertFiles[] = {
     "/etc/ssl/certs/ca-certificates.crt", "/etc/pki/tls/certs/ca-bundle.crt",
     "/etc/ssl/ca-bundle.pem", "/etc/pki/tls/cacert.pem",
     "/etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem"};
-const char* kLinuxCertDirectories[] = {
+const char* kCertDirectories[] = {
     "/etc/ssl/certs", "/system/etc/security/cacerts", "/usr/local/share/certs",
     "/etc/pki/tls/certs", "/etc/openssl/certs"};
+#endif // GPR_LINUX || GPR_ANDROID
+#ifdef GPR_FREEBSD
```

Comment:
What's the difference between using `ifdef GPR_FREEBSD` and `if defined(GPR_FREEBSD)`? 
We probably want to keep the syntax consistent.

Reply:
Thanks for taking a look! So `ifdef VAL` and `if defined(VAL)` are equivalent, the reason why `if defined(VAL)` is used above is that it allows multiple conditions to be combined into a single if statement (i.e. `ifdef GPR_LINUX || GPR_FREEBSD` would not work). See https://stackoverflow.com/questions/39290019/is-if-defined-macro-equivalent-to-ifdef-macro

Let me know if you'd still like me to change it, happy to do it if you think it's cleaner that way.

re: Mac PR: I haven't sent it yet because I'm wondering whether we should just rename this file, remove the "_linux" suffix, and add the Mac-specific path in here too, for the same reason why @drfloob suggested merging the FreeBSD and Linux implementations. That way we would only have this implementation for all platforms where we support loading system roots, and the fallback implementation for all other platforms. WDYT? If that sounds good to you I'll send in a follow-up PR once this is merged (unless you think it's fine to add both here).

Reply:
I see, thanks for the explanation! Would you mind change this to `if defined(VAL)` to make it consistent?